PROTOCOL.md

```diff
@@ -861,6 +890,7 @@ Checkpoint files must be written in [Apache Parquet](https://parquet.apache.org/
  * The [metadata](#Change-Metadata) of the table
  * Files that have been [added and removed](#Add-File-and-Remove-File)
  * [Transaction identifiers](#Transaction-Identifiers)
+ * [Domain Metadata](#Domain-Metadata)
```

Comment:
According to https://github.com/delta-io/delta/issues/1741, this adds a new feature. should we update https://github.com/delta-io/delta/blob/master/PROTOCOL.md?

Reply:
Oh, you're right it does need a table feature spec

Reply:
Good point I added an `Enablement` section to describe the writer feature requirement etc.